package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/identity-components",
-  "version": "5.0.3",
+  "version": "5.0.4",
```

Comment:
Please back this version bump out of the pr. Thanks.

Reply:
Version bump rolled back.